services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/Incident.java

```diff
@@ -466,6 +473,12 @@ public abstract static class Builder extends DirectionsJsonObject.Builder<Builde
      */
     public abstract Builder numLanesBlocked(@Nullable Integer numLanesBlocked);
 
+    /**
+     * Sets list of roads names affected by the incident.
+     * @param affectedRoadNames list of roads names affected by the incident.
+     */
+    public abstract Builder affectedRoadNames(@Nullable List<String> affectedRoadNames);
```

Comment:
```suggestion
     */
     @NonNull
    public abstract Builder affectedRoadNames(@Nullable List<String> affectedRoadNames);
```

We can also add the above missing annotations 👍